### PR TITLE
[sdg] homepage update

### PR DESCRIPTION
**Pull request by chejennifer**
- update homepage according to mocks

Screenshot 2023-09-06 at 4 01 54 PM
Screenshot 2023-09-06 at 4 07 08 PM

Screenshot 2023-09-06 at 4 02 18 PM

### CODE REVIEW

File: experimental/sdg-tracker/src/components/home/ExploreSection.tsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    <Container>
+      <h3>Explore UN Data Commons for the SDGs</h3>
+      <SearchBarContainer>
+      <SearchBar
+        initialQuery="What is the global poverty rate?"
+        isSearching={false}
+        onSearch={(q) => {
+          const searchParams = new URLSearchParams();
+          searchParams.set(QUERY_PARAM_QUERY, q);
+          window.open(`/search?${searchParams.toString()}`, "_self")
```
Comment (dwnoble) [inline experimental/sdg-tracker/src/components/home/ExploreSection.tsx:102]: use `useHistory` & `history.push` from `react-router-dom` here instead

File: experimental/sdg-tracker/src/components/home/GoalSection.tsx [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+          {
+            goalSection.map((goal: GoalItem) => {
+              return (<div className="goal-item" key={goal.topicDcid} onClick={() => window.open(`/goals/dc/topic/sdg_1?v=${goal.topicDcid}`, "_self")}>
+                <div style={{backgroundColor: goal.color}} className="goal-number">{goal.goalNum}</div>
+                <div className="goal-content"><div className="goal-name">{goal.name}</div><div className="goal-icon"><img src={goal.iconUrl}/></div></div>
+              </div>)
+            })
+          }
+          {idx === 1 && (
+            <div className="goal-item" onClick={() => window.open(`/goals`, "_self")}>
```
Comment (dwnoble) [inline experimental/sdg-tracker/src/components/home/GoalSection.tsx:148]: ditto

File: experimental/sdg-tracker/src/components/home/GoalSection.tsx [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+        <div className="title">Explore SDG Data by Goal</div>
+        <div className="line-separator" />
+        <div className="description">Learn about SDG progress across all 17 Goals -- with data, insights and infographics in one place for a comprehensive overview.</div>
+      </HeaderContainer>
+      <GoalContainer>{
+        goalSections.map((goalSection: GoalItem[], idx) => {
+          return (<div className="goal-section" key={`section-${idx}`}>
+          {
+            goalSection.map((goal: GoalItem) => {
+              return (<div className="goal-item" key={goal.topicDcid} onClick={() => window.open(`/goals/dc/topic/sdg_1?v=${goal.topicDcid}`, "_self")}>
```
Comment (dwnoble) [inline experimental/sdg-tracker/src/components/home/GoalSection.tsx:141]: `history.push`

File: experimental/sdg-tracker/src/components/home/PlaceSection.tsx [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+        notFoundContent="No results found"
+        filterOption={(inputValue, option) =>
+          option!.value.toUpperCase().indexOf(inputValue.toUpperCase()) !==
+            -1 ||
+          option!.dcid.toUpperCase().indexOf(inputValue.toUpperCase()) !== -1
+        }
+        onChange={(value, option) => {
+          setValue(value);
+          if ("dcid" in option) {
+            window.open(`/countries?p=${option.dcid}`, "_self")
```
Comment (dwnoble) [inline experimental/sdg-tracker/src/components/home/PlaceSection.tsx:136]: `history.push`

File: experimental/sdg-tracker/src/config/homeGoals.json [hunk 5]
```diff
@@ -0,0 +1,121 @@
+[
```
Comment (dwnoble) [inline experimental/sdg-tracker/src/config/homeGoals.json:1]: Can you combine this with the existing `rootTopics.json`?

**Comment by dwnoble** [comment]
This is looking great, Jennifer!

Some mobile overflow finds (iphone 12 pro screen size)
Screenshot 2023-09-06 at 4 34 51 PM

Screenshot 2023-09-06 at 4 34 59 PM

File: experimental/sdg-tracker/src/config/homeGoals.json (hunk 5 above)
Comment (beets) [reply experimental/sdg-tracker/src/config/homeGoals.json:1]: +1  -- we could use this elsewhere too.

**Review by beets — APPROVED** [review]
this is looking nice! approving pending resolved comments..

File: experimental/sdg-tracker/src/components/home/ExploreSection.tsx (hunk 1 above)
Comment (chejennifer) [reply experimental/sdg-tracker/src/components/home/ExploreSection.tsx]: done

File: experimental/sdg-tracker/src/components/home/GoalSection.tsx (hunk 2 above)
Comment (chejennifer) [reply experimental/sdg-tracker/src/components/home/GoalSection.tsx]: done

File: experimental/sdg-tracker/src/components/home/GoalSection.tsx (hunk 3 above)
Comment (chejennifer) [reply experimental/sdg-tracker/src/components/home/GoalSection.tsx]: done

File: experimental/sdg-tracker/src/components/home/PlaceSection.tsx (hunk 4 above)
Comment (chejennifer) [reply experimental/sdg-tracker/src/components/home/PlaceSection.tsx]: done

File: experimental/sdg-tracker/src/config/homeGoals.json (hunk 5 above)
Comment (chejennifer) [reply experimental/sdg-tracker/src/config/homeGoals.json]: done

**Comment by chejennifer** [comment]
thanks for the reviews

**Comment by dwnoble** [comment]
Thank you! last thing found some mobile overflow in the top section

Screenshot 2023-09-06 at 5 23 34 PM

**Pull request merged**